BraveShared/Favicons/FaviconRenderer.swift

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import BraveCore
+
+/// A class for rendering a FavIcon onto a `UIImage`
+class FaviconRenderer {
```

Comment:
You have multiple classes that do the same exact setup and execution (`BundledFaviconImageRenderer`, `FaviconRenderer`), could this not be simplified or consolidated?

Reply:
They do different things. 
One class is for rendering Favicons from Brave-Core only (including monogram images).
The other class is for loading and rendering Favicons that are Bundled with the app (TopSites), no monograms at all.

Reply:
I know they do different things, I said that they have the exact same setup + execution, in fact the two files are mostly duplicated aside from the body of the task work and the name of the load method (which could be the same).